crypto/rand/rand_win.c

```diff
@@ -567,51 +166,6 @@ int RAND_poll(void)
     return (1);
 }
 
-int RAND_event(UINT iMsg, WPARAM wParam, LPARAM lParam)
```

Comment:
I think this is a good time to make a clean break.  But keeping them as deprecated would be a reasonable option.